foerder.html

```diff
@@ -226,12 +226,18 @@
 				formDict[pair[0]] = pair[1];
 			}
 
-			var jsonMsg = JSON.stringify(formDict);
```

Comment:
Yep, that's a problem I'm still thinking about. Probably much safer to use json and write a tool which sanitizes the input and then writes it to the database directly. But we definitely need to write smaller json then. And put up a new project for the input side.